config/moved_config.go

```diff
@@ -147,6 +148,12 @@ func (cfg *Config) validateService() error {
 			}
 		}
 	}
+
+	// Check when service telemetry metric level is not none, the metrics address should not be empty
+	if cfg.Service.Telemetry.Metrics.Level != configtelemetry.LevelNone && cfg.Service.Telemetry.Metrics.Address == "" {
+		return fmt.Errorf("collector telemetry metric address should exist when metric level is not none")
+	}
+
```

Comment:
[Idea]: On `cfg.Service.Telemetry` add a `Validation` func and call that one.

What do you think?

Reply:
I saw other config package use `Validate`, not `Validation`, I think I better keep up?

@bogdandrutu

Reply:
@bogdandrutu I already changed this with `Validate`

Reply:
yes, `Validate` sorry.